### PR TITLE
overlook some plugins

### PR DESCRIPTION
**Pull request by cgzones**
Stable pr of #1009

### CODE REVIEW

File: plugins/node.d.linux/proc_pri.in [hunk 1]
```diff
@@ -60,8 +67,8 @@ if [ "$1" = "config" ]; then
 fi
 
 echo -n "high.value "
-ps --no-header -eo stat | grep \< | wc -l
+$PS --no-header -eo stat | grep \< | wc -l
```
Comment (sumpfralle) [inline plugins/node.d.linux/proc_pri.in]: You could quote `$PS` here.

Comment (cgzones) [reply plugins/node.d.linux/proc_pri.in]: done

**Comment by cgzones** [comment]
I looked over the scripts and they should be ready for `set -e`. `set -u` is problematic cause `if [ "$1" = "autoconf" ]; then` is used widely.

**Comment by sumpfralle** [comment]
>  looked over the scripts and they should be ready for `set -e`.

Great!

> `set -u` is problematic cause `if [ "$1" = "autoconf" ]; then` is used widely.

Yes, indeed - this needs some more changes in a few places.

Going forward step by step - thank you!

**Pull request merged**